addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -751,9 +751,34 @@ blockquote {
     }
 }
 %o-we-background-layer {
-    @include o-position-absolute(0, 0, 0, 0);
+    // Position the layer on top of the parent
     position: absolute !important;
+
+    // Place it at the center, independently from the way its size is computed
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Good catch :+1: (don't know how I did not see because I did test the shapes, but it's not something which affects all shapes in all configurations).

It's actually the system of @bvr-odoo (`ZoomedBackgroundShape`) which adds a very small offset on the sides to try and prevent the "white line" bug we always had with shapes. We should really just review/retest it: it creates bugs... the question is "does it solve more than the bug it creates"? Or maybe it should be even better and only affect shapes which are connected to another one to avoid creating bugs in "alone shapes". Or maybe the offset value has to be reviewed.

Here is an example:

Before my branch: if you have good eyes, you'll see a black line on the right of the shape
![image](https://github.com/user-attachments/assets/1b683f56-75cc-4feb-9044-4c5289cd456d)

With my branch (after my fix I did not push yet): you'll see an image line on the **left** of the shape (unfortunately it is more visible with my branch here... but maybe because of the image I chose?).
![image](https://github.com/user-attachments/assets/2025be32-55d1-4941-8d77-1751f4e6ed21)

I suggest we merge with my change and re-evalutes BVR's code afterwards. Maybe that with the new way I am positioning those shapes, his code is not needed anymore who knows. Or maybe it is even more needed and we'll find a way to improve it.

Reply:
Indeed, unfortunately, I have no solution then. %-related centering is messing things up. Let's leave this open and come back to it later. Meanwhile, we'll remove the overflow: hidden anyway. Keeping the same bug we have in stable at the moment. I'll open another PR with that.